tests/test_parallel_offset.py

```diff
@@ -1,31 +1,32 @@
 from . import unittest
 from shapely.geometry import LineString, LinearRing
+from shapely.testing import assert_geometries_equal
 from shapely.wkt import loads
 
 class OperationsTestCase(unittest.TestCase):
 
     def test_parallel_offset_linestring(self):
         line1 = LineString([(0, 0), (10, 0)])
         left = line1.parallel_offset(5, 'left')
-        self.assertEqual(left, LineString([(0, 5), (10, 5)]))
+        assert_geometries_equal(left, LineString([(0, 5), (10, 5)]))
         right = line1.parallel_offset(5, 'right')
-        self.assertEqual(right, LineString([(10, -5), (0, -5)]))
+        assert_geometries_equal(right, LineString([(10, -5), (0, -5)]), normalize=True)
```

Comment:
The new implementation returned the coordinates in a different order, for which I added `normalize=True` to get this passing.